### PR TITLE
Install instructions

### PR DESCRIPTION
**Pull request by renefritze**

### CODE REVIEW

**Comment by pmli** [comment]
Is there a reason for recommending wheels instead of `pip install -r requirements.txt`?

I would be for adding binder links (or at least mentioning them) in the root `README.md` file.

**Comment by renefritze** [comment]
> Is there a reason for recommending wheels instead of `pip install -r requirements.txt`?

Installation from the wheel does not need a compiler setup. We can add an additional "plain" pip section though.
> 
> I would be for adding binder links (or at least mentioning them) in the root `README.md` file.

RIght.

**Comment by renefritze** [comment]
How about the conda stuff now, @pmli ? Should we leave the conda package install out of the install guide for now?

**Comment by pmli** [comment]
> How about the conda stuff now, @pmli ? Should we leave the conda package install out of the install guide for now?

Yes, I don't know how to do it properly, so I would be for leaving it out for now.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,72 @@
...
+# now open the displayed URL (127.0.0.1) in your browser
+```
+
+### myBinder.org
+
+If cannot or do not want to install pymMOR you can play with the exercises and tutorials using
+the mybinder.org infrastructure:
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/pymor/2020_online_course/main)
+
+Please note that this a free servive with usage limitations (no persistent data, compute capacity, concurrent users),
```
Comment (pmli) [inline README.md]:
```suggestion
Please note that this a free service with usage limitations (no persistent data, compute capacity, concurrent users),
```

**Pull request merged**